core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -635,6 +635,14 @@ object TestUtils extends Logging {
     builder.build()
   }
 
+  /**
+   * Generate a random string of letters and digits of the given length
+   *
+   * @param len The length of the string
+   * @return The random string
+   */
+  def randomString(len: Int): String = JTestUtils.randomString(len)
```

Comment:
nit: do we need this or can work with existing `randomBytes`?

Reply:
Didn't realise the existence of randomBytes. Made the amends, thanks for the review